lib/dialogs/buy/bloc/BuyDialogBloc.dart

```diff
@@ -51,16 +52,25 @@ class BuyDialogBloc extends Bloc<BuyDialogEvent, BuyDialogState> {
       } else {
         final errorMsg = response.getRight().toNullable()!.errorMsg;
         //TODO Create User facing error messages https://athletex.atlassian.net/browse/AX-466
-        print(errorMsg);
+        print(errorMsg.toString() + ' This is mymessage 3');
```

Comment:
Remove print statements where it was added

Reply:
definitely need to clean this up